src/widget/container/mod.rs

```diff
@@ -155,16 +160,21 @@ where
         };
 
         for child in &self.options.children {
-            let default_size = match (self.options.alignment, self.main_axis()) {
+            let child_alignment = child
+                .layout_options()
+                .alignment
+                .unwrap_or(self.options.alignment);
+
+            let default_size = match (child_alignment, self.main_axis()) {
```

Comment:
This is the only behavior change in the code, the rest is just moving things around. (Seems suspiciously easy?)

Reply:
Makes sense conceptually that the child can "override" the parent by supplying its own configuration value